uiowa/uiowa
# Added credentials to the author byline in article's author fields

Resolves: https://github.com/uiowa/uiowa/issues/5171

[The trending topics section of the Mental Health at Iowa site](https://mentalhealth.uiowa.edu/trending-topics) has been populated with people that have their credentials stored within their last name. I've been publishing articles for them since April and I've been putting credentials in their last names just to keep them visible in the byline and consistent. It's become apparent that because we're in academia these credentials are important to the authors of posts and I've grown weary of putting them in incorrectly.

This PR adds a person's credentials to the article bylines that appear in article list blocks and on the article itself. 

# How to test

1. Stay on the `main` branch
2. Pull down the Mental Health at Iowa site: `blt ds --site=mentalhealth.uiowa.edu` and login: `drush @mentalhealth.local uli`
3. [Edit Gerard Clancy's person page](https://mentalhealth.uiowa.ddev.site/node/501/edit) to give them proper credentials in the "Credentials" field and remove their improper "last name credentials."
4. Return to the homepage and see Gerard Clancy's credentials are **NOT** showing up in the first post in the "Topics in Mental Health" article list block: <img width="100%" alt="Screen Shot 2022-09-29 at 10 56 49 AM" src="https://user-images.githubusercontent.com/472923/193080386-483a46ab-014a-4070-8692-6ef6ab730197.png">
5. Also note that upon clicking the article, their credentials do not show up in the article itself: <img width="100%"  alt="Screen Shot 2022-09-29 at 10 57 56 AM" src="https://user-images.githubusercontent.com/472923/193080728-4d239a7c-7d7e-4b58-b0ed-9406749c0312.png">
6. Check out this branch `fix_article_author_credentials` and clear your cache: `drush @mentalhealth.local cr`
7. Return to the homepage and see Gerard Clancy's proper credentials now appear in the first post in the "Topics in Mental Health" article list block: <img width="100%"  alt="Screen Shot 2022-09-29 at 10 53 58 AM" src="https://user-images.githubusercontent.com/472923/193079677-ec6873dc-802f-466b-8fb9-3e026eabf72b.png">
8. Also note that their credentials now show up in the article itself: <img width="100%"  alt="Screen Shot 2022-09-29 at 10 59 36 AM" src="https://user-images.githubusercontent.com/472923/193081049-852a1cab-62ab-4755-a8fb-77c386943c60.png">
